web/management/commands/create_test_data.py

```
@@ -574,4 +578,149 @@ def handle(self, *args, **kwargs):
+                    slug=slugify(subject_name),
+                    description=subject_data.get("description", f"Study of {subject_name}"),
+                    icon=subject_data.get("icon", "fas fa-graduation-cap"),
+                    order=len(subjects) + 1,
+                )
+                subjects.append(subject_obj)
+                self.stdout.write(f"Created new subject: {subject_obj.name}")
+
+            # Create memes for this subject
+            for meme_info in subject_data["memes"]:
+                # Select a random user as uploader
+                uploader = random.choice(users)
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)